### DISCUSSION TITLE
Nov Newsletter - Cloudberry Database (2023/11)

### DISCUSSION THREAD

**Discussion by tuhaihe** [apache/cloudberry#319] [Announcements]
Hello there! Thank you for joining us for the November newsletter of the Cloudberry Database project! 

We are thrilled to have you as a part of our community. In this newsletter, we will be sharing the latest updates, improvements, and contributions made by our incredible members. We have a lot to talk about, from exciting new features to bug fixes, so let's dive right in!

## Highlight: Cloudberry Database Bootcamp

We recently announced the [Cloudberry Database Bootcamp](https://github.com/cloudberrydb/bootcamp) program, a comprehensive program that provides a hands-on experience with Cloudberry Database. It is tailored to help you understand Cloudberry Database's capabilities and features a range of materials, including tutorials, sample code, and crash courses. If you're new to Cloudberry Database, this bootcamp is the perfect place to begin. You can check the [announcement discussion](https://github.com/orgs/cloudberrydb/discussions/299) for more details. 

<img width="800" alt="cbdb-sandbox" src="https://github.com/cloudberrydb/cloudberrydb/assets/1284465/089ea6e0-9117-43c9-b753-73efb0878f26">

## Pull Requests

- #315 by @hw118118
- #314 by @wenchaozhang-123
- #311 by @roseduan
- #309 by @lss602726449
- #308 by @smartyhero
- #306 by @Ray-Eldath
- #304 by @HuSen8891
- #300 by @tuhaihe
- #298 by @avamingli
- #297 by @MisterRaindrop
- #296 by @roseduan
- #294 by @avamingli
- #293 by @avamingli
- #292 by @avamingli
- #291 by @Ray-Eldath
- #290 by @Zhangbaowen-Hashdata
- #289 by @avamingli
- #288 by @avamingli
- #287 by @tuhaihe
- #286 by @wenchaozhang-123
- #284 by @avamingli
- #282 by @Ray-Eldath

## Issues

- #317 by @congxuebin
- #307 by @avamingli
- #302 by @Ray-Eldath
- #301 by @Ray-Eldath
- #295 by @congxuebin
- #273 by @congxuebin
- #285 by @Ray-Eldath
- #281 by @Ray-Eldath
- #279 by @Light-City

## Website

Currently, we are working on the new version of the Cloudberry Database website, which will have a fresh and modern design in the upcoming weeks. Additionally, we are diligently working on the Cloudberry Database user document. Stay tuned for more exciting news! 

## Contributors

🎈️🎊️ Thanks to the following contributors for helping make Cloudberry Database better this month:

> @wenchaozhang-123, @hw118118, @roseduan, @lss602726449, @smartyhero, @HuSen8891, @Ray-Eldath, @tuhaihe, @avamingli, @MisterRaindrop, @Zhangbaowen-Hashdata, @congxuebin, @Light-City, @ginobiliwang, @TomShawn, @RyanWei, @liang8283

## Join us

The Cloudberry Database community welcomes everyone to contribute, regardless of their level of experience. We encourage all types of contributions, no matter how small. Our [contribution guide](https://cloudberrydb.org/contribute/how-to-contribute) is available to help you get started with the process.

In addition, we offer various channels for community members to discuss, seek help, provide feedback, and chat. You can find support [here](https://cloudberrydb.org/support).

Join us and be part of our community!

## Past Newsletters

- #274
- #225
- #181

-------

[![Slack](https://img.shields.io/badge/Join_Slack-6a32c9)](https://communityinviter.com/apps/cloudberrydb/welcome) [![Twitter Follow](https://img.shields.io/twitter/follow/cloudberrydb)](https://twitter.com/cloudberrydb) [![Website](https://img.shields.io/badge/Visit%20Website-eebc46)](https://cloudberrydb.org)

**Comment by TomShawn** [comment]
Hooray!
